runtime/doc/repeat.txt

```diff
@@ -172,8 +172,7 @@ Using Vim scripts					*using-scripts*
 For writing a Vim script, see chapter 41 of the user manual |usr_41.txt|.
 
 					*:so* *:source* *load-vim-script*
-:so[urce] {file}	Read Ex commands from {file}.  These are commands that
-			start with a ":".
+:so[urce] {file}	Runs vim or lua {file}
```

Comment:
perhaps mention if there is no extension, the file is assumed to be vim script?

Reply:
done